llvm/lib/Analysis/InlineCost.cpp

```diff
@@ -1943,7 +1943,7 @@ void InlineCostCallAnalyzer::updateThreshold(CallBase &Call, Function &Callee) {
   // and the callsite.
   int SingleBBBonusPercent = 50;
   int VectorBonusPercent = TTI.getInlinerVectorBonusPercent();
-  int LastCallToStaticBonus = InlineConstants::LastCallToStaticBonus;
```

Comment:
We still can't delete `InlineConstants::LastCallToStaticBonus` because it is used in another place where there is no access to TTI (including its parent function).

https://github.com/llvm/llvm-project/blob/4169338e75cdce73d34063532db598c95ee82ae4/llvm/lib/Analysis/InlineAdvisor.cpp#L323